vnext/Microsoft.ReactNative.Managed.CodeGen/Microsoft.ReactNative.Managed.CodeGen.csproj

```diff
@@ -68,9 +68,10 @@
   </Target>
 
 
-   <!-- Override -->
+  <!-- Override -->
   <Target Name="_CheckForMismatchingPlatform">
 
   </Target>
 
+  <Target Name="Deploy" />
```

Comment:
@dannyvv note that making Deploy not do anything means you can't run this in Debug (since we'll try to deploy from layout in Debug and the target isn't doing anything). I thought we had made this and other projects include Microsoft.ReactNative.Uwp.CSharpApp.targets  and Microsoft.ReactNative.Uwp.CppApp.targets?

Reply:
It should not be needed here, Jon made the suggestion to do so. 
This is a regular netcore app that does the code generation for C#. It is never prject referenced by any uwp or react app. I just added it here for safety in case the solution geenrated metabuild proejct does funny stuff.

Reply:
@dannyvv so to make sure, you can `react-native run-windows` this project and it deploys ok?

Reply:
@dannyvv One thing we could do instead of requiring everybody to define an empty Deploy target is to have the target in some RNW prop file which then could be overwritten when a project includes the right targets; it would be unorthodox to have a target in a props but it would prevent the tax of having empty Deploy targets everywhere

Reply:
I would recommend waiting until we are shipping binaries, ando/or refactored the layout a bit. This is an okay workaround for now.